Dockerfile.django

```diff
@@ -89,14 +89,12 @@ USER root
 RUN \
     adduser --system --no-create-home --disabled-password --gecos '' \
         --uid ${uid} ${appuser} && \
-    chown -R ${appuser} /app && \
-    chmod 0700 /app && \
-    chmod 0750 -R /app/* && \
-	chmod g=u /app && \
-	chmod -R g=u /app/* && \
+    chown -R root:root /app && \
+    chmod -R u+rwX,go+rX,go-w /app && \
     mkdir /var/run/${appuser} && \
     chown ${appuser} /var/run/${appuser} && \
-	chmod g=u /var/run/${appuser}
+	  chmod g=u /var/run/${appuser} && \
+    mkdir -p media && mkdir -p media/threat && chown -R ${uid} media
```

Comment:
Wouldn't `mkdir -p media/threat be sufficient here? I believe the p flag creates all intermediary directories if they do not exist.

Reply:
Yep, it can be single mkdir